### ISSUE TITLE
Bazel build hang or approximative time to build Bazel?

### ISSUE THREAD

**Issue by calocedrus** [samjabrahams/tensorflow-on-raspberry-pi#107]
I'm currently compiling bazel following the nice guide here. But `./compile.sh` seemed stuck at
`Building Bazel from scratch ..` (only 2 dots `..`) for about 30mn with only 25% CPU loading. Thinking it had hanged, I `ctrl-c` bazel compile. Does the build process show its progression? How long does it take to build (even approximately)?

* Raspberry Pi model: 3
* Operating System used: Jessie Light, updated, upgraded (not dist-upgrade) 
* Version of Python used: 3.6.1 compiled from source
* SD card memory size: 16GB
* Size of USB/other device used as swap (if building from source): 16GB

**Comment by calocedrus** [comment]
I eventually let the process do its job, after about 5hours, it told me bazel failed to build. I'll report it in another issue.

**Comment by eraoul** [comment]
Yes, the build process shows progression: you should eventually see an update such as (300 / 1400) files built. It's not visible at all points of the build process, but it's often there.

It took me around 1 or 1.5 hours to build on Rasp PI 3b, with python 3.4, 16BG SD card, 8GB swap.

**Comment by topicsearch** [comment]
this is the final line output in the console after something such as (1700/1756) done.
INFO: 1756 processes: 1756 local.
INFO: Build completed successfully, 1862 total actions
